tutorials/shading/shading_reference/shading_language.rst

```diff
@@ -187,6 +187,65 @@ Keep in mind that some architectures (mainly mobile) benefit a lot from this, bu
 to find out more. In all honesty though, mobile drivers are buggy, so, to stay out of trouble, make simple 
 shaders without specifying precision unless you *really* need to.
 
+Arrays
+------
+
+Arrays are containers for the multiple variables of similar type. In GLSL they can be declared local to function, global space or uniform.
+Note: At the time of 3.2 only local arrays has been implemented.
+
+Local arrays
+~~~~~~~~~~~~
+
+Local arrays are declared in the functions. They can use all of the allowed datatypes, except the samplers.
+Declaration of the array is follow the C syntax : 'typename + identifier + [array size]'.
```

Comment:
The array declaration follows a C-style syntax: ``typename + identifier + [array size]``.